src/main/resources/assets/js/application/common/directives/tdprNewPasswordDirective.js

```diff
@@ -0,0 +1,47 @@
+define(['angular', 'application/common/tdprCommonModule'], function (angular, tdprCommonModule) {
+    tdprCommonModule.directive("newPassword", function () {
+        return {
+            restrict: 'AE',
+            templateUrl: '/html/partials/common/tdpr-directive-new-password.html',
+            scope: {
+                isPasswordValid: '='
+            },
+            link: function (scope) {
+                scope.arePasswordsDifferent = false;
+                scope.isPasswordValid = true;
+                scope.newPassword = '';
+                scope.confirmPassword = '';
+
+                scope.$watch('[newPassword, confirmPassword]', function (newValue, oldValue, scope) {
+                    scope.arePasswordsDifferent = newValue[0] === newValue[1] ? false : true;
```

Comment:
The ternary operator is redundant here and can be replaced simply with `newValue[0] !== newValue[1]`.